tasks/testing.rake

```diff
@@ -1,4 +1,8 @@
+require_relative "#{__dir__}/../lib/opal/os"
```

Comment:
nit: also here using __dir__ shouldn't be necessary

Reply:
I had to use __dir__ so that it would use the patched version